src/browser/modules/BrowserDOMWindow-sys-mjs.patch

```diff
@@ -1,8 +1,64 @@
 diff --git a/browser/modules/BrowserDOMWindow.sys.mjs b/browser/modules/BrowserDOMWindow.sys.mjs
-index 2bc4918c15b5ea80e486db66d20bb9668b9e29c0..3800a3807f11c5f65ce0d602a775f95037f5dcf8 100644
+index 2bc4918c15b5ea80e486db66d20bb9668b9e29c0..c73cc27d443623b935c8243ed165acc399571bb6 100644
 --- a/browser/modules/BrowserDOMWindow.sys.mjs
 +++ b/browser/modules/BrowserDOMWindow.sys.mjs
-@@ -362,7 +362,7 @@ export class BrowserDOMWindow {
+@@ -42,7 +42,6 @@ export class BrowserDOMWindow {
+    * @type {Window}
+    */
+   win = null;
+-
+   constructor(win) {
+     this.win = win;
+   }
+@@ -151,14 +150,45 @@ export class BrowserDOMWindow {
+       console.error("openURI should only be called with a valid URI");
+       throw Components.Exception("", Cr.NS_ERROR_FAILURE);
+     }
++
++    // Zen Air Traffic Control integration - intercept external links
++    if (aURI && (aFlags & Ci.nsIBrowserDOMWindow.OPEN_EXTERNAL)) {
++      try {
++        const { ZenAirTrafficControlIntegration } = ChromeUtils.importESModule(
```

Comment:
Since it's already imported, we could run `this.win.ZenAirTrafficControlIntegration` instead of re-parsing the module